src/main/java/feedmysheep/feedmysheepapi/domain/cell/app/repository/CellGatheringRepository.java

```diff
@@ -18,4 +20,12 @@ public interface CellGatheringRepository extends JpaRepository<CellGatheringEnti
   @Query("SELECT cg FROM CellGatheringEntity cg WHERE cg.cellGatheringId = :cellGatheringId and cg.isValid = true")
   CellGatheringEntity getCellGatheringByCellGatheringId(
       @Param("cellGatheringId") Long cellGatheringId);
+
+  // TODO 테스트코드 작성
+  @Transactional
+  @Modifying
+  @Query("UPDATE CellGatheringEntity cg SET cg.isValid = false,cg.updatedBy = :memberId WHERE cg.cellGatheringId = :cellGatheringId")
```

Comment:
```suggestion
  @Query("UPDATE CellGatheringEntity cg SET cg.isValid = false, cg.updatedBy = :memberId WHERE cg.cellGatheringId = :cellGatheringId")
```

option+command+L 계속 눌러주고 있는거죠..? 다른 파일들도 칸 안 맞는 것들이 많던뎅